Sources/BlogModule/Controllers/BlogPostController.swift

```diff
@@ -65,3 +65,38 @@ struct BlogPostController: FeatherController {
         model.title
     }
 }
+
+/// Overide default Route Builder
+extension BlogPostController {
+    
+    func listApiTimestamp(_ req: Request) throws -> EventLoopFuture<Response> {
+        let start: Date = req.query["start"] ?? Date(timeIntervalSince1970: 0)
+        let end: Date = req.query["end"] ?? Date()
+        let paginated =  listLoader
+            .paginate(req, start: start, end: end, withDeleted: true).map { pc -> PaginationContainer<ListApi.ListObject> in
+                let api = ListApi()
+                let items = pc.map { api.mapList(model: $0) }
+                return items
+            }
+        var headers = HTTPHeaders()
```

Comment:
@tib Quick question here. I was actually trying to access the count of items (or any other info that are actually paginated details (like page, limit etc...). The idea is that instead of using GET, you could use HEAD (which work by default with Vapor if you implement GET) and have an empty body but all detail in the header.

Problem, is that those value are Future. Any idea how to modify the Response Header in the loop?

Reply:
That's what I was originally doing with my PHP code, https://github.com/charlymr/IRLServerConnection/blob/e229991622c69de1bf253de1373d4425546fe9c5/Sources/IRLServerConnection/Helper/ManagedObjectServerMaping%2BEntity.swift#L129 
But here I had to change a bit strategy. Just an idea

Reply:
Will check this later on.